samples/nrf9160/download/src/main.c

```diff
@@ -121,7 +121,7 @@ static int callback(const struct download_client_evt *event)
 		}
 
 #if CONFIG_SAMPLE_COMPUTE_HASH
```

Comment:
Maybe add `CONFIG_SAMPLE_COMPUTE_HASH` to extra configs in sample.yaml, to get a better build coverage?

Reply:
Sounds good! Added it now.

Reply:
Thanks!